.circleci/config.yml

```diff
@@ -55,25 +23,87 @@ jobs:
             ./architect deploy
           fi
 
+
+
   e2eTestCurrentVersionPR:
     environment:
+      MINIKUBE_VERSION: v0.25.0
       TESTED_VERSION: "current"
-    <<: *e2eTest
+    machine: true
+    working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/aws-operator
+    steps:
+    - checkout
+    - run: |
+        wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2e-harness/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+        chmod +x ./e2e-harness
+    - run: ./e2e-harness localkube
+    - run: ./e2e-harness setup --remote=false
+    - run: ./e2e-harness test --test-dir=integration
```

Comment:
I want to split tests in the future so already making this explicit so we can just rename folders later on.

Reply:
I think this repetition of code doesn't help now. IMO it would be better to do it when needed, that is, when the tests are split. Otherwise, this change only adds maintenance burden, we need to change things in four different places instead of one.

Reply:
As I already mentioned I was not able to reuse the YAML block before while passing env vars into it. I think I might doing something wrong. Just don't know what.

Reply:
Could you please explain the problem a little bit more? We previously had something like:
```
e2eTestCurrentVersionMaster:
     environment:
       TESTED_VERSION: "current"
     <<: *e2eTest
```
and the env vars in the `environment` section were available in the inserted block. What is not working now?, why did this stop working?

Reply:
Why it did stop working is the question. While I was testing I noticed the `TESTED_VERSION` is not properly set and thus not available in the golang integration test. I got the panic where the env is read.

Reply:
mm it should work then, let me give it a try

Reply:
Sure, go ahead.

Reply:
These changes work fine https://github.com/giantswarm/aws-operator/pull/836 already proposed to merge here. Also added a missing `attach_workspace`.

Reply:
Cool thing. Lets get it in.